spk/rutorrent/src/installer.sh

```diff
@@ -207,24 +259,67 @@ preupgrade ()
     rm -fr ${TMP_DIR}/${PACKAGE}
     mkdir -p ${TMP_DIR}/${PACKAGE}
     mv ${WEB_DIR}/${PACKAGE}/conf/config.php ${TMP_DIR}/${PACKAGE}/
+    if [ -f "${WEB_DIR}/${PACKAGE}/.htaccess" ]; then
```

Comment:
If someone defines a `.htaccess` to configure authentication you'd better not lose it when upgrading :)